pkg/controller/pxc/backup.go

```diff
@@ -28,6 +31,30 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileBackups(cr *api.PerconaXtraDBCl
 	if cr.Spec.Backup != nil {
 		bcpObj := backup.New(cr)
 
+		if cr.Status.Status == api.AppStateReady && cr.Spec.Backup.PITR.Enabled {
+			binlogCollector, err := deployment.GetBinlogCollectorDeployment(cr)
+			if err != nil {
+				return fmt.Errorf("get binlog collector deployment for cluster '%s': %v", cr.Name, err)
+			}
+			binlogCollectorName := cr.Name + "-pitr"
+			currentCollector := appsv1.Deployment{}
+			err = r.client.Get(context.TODO(), types.NamespacedName{Name: binlogCollectorName, Namespace: cr.Namespace}, &currentCollector)
+			if err != nil && k8serrors.IsNotFound(err) {
+				err = r.client.Create(context.TODO(), &binlogCollector)
+				if err != nil && !k8serrors.IsAlreadyExists(err) {
+					return fmt.Errorf("create binlog collector deployment for cluster '%s': %v", cr.Name, err)
+				}
+			} else if err != nil {
+				return fmt.Errorf("get binlogCollector '%s': %v", binlogCollectorName, err)
+			} else {
+				currentCollector.Spec = binlogCollector.Spec
+				err = r.client.Update(context.TODO(), &currentCollector)
+				if err != nil {
+					return fmt.Errorf("update binlogCollector '%s': %v", binlogCollectorName, err)
+				}
+			}
+		}
+
```

Comment:
what if PITR was enabled and disabled in this particular moment.
do we delete deployment?